src/ReflectionParameter.php

```diff
@@ -207,13 +214,14 @@ public function getClass()
                     return $this->getDeclaringClass()->getParentClass();
                 }
 
-                throw new ReflectionException("Can not resolve a class name for parameter");
+                return null;
             }
             $className = $parameterType->toString();
 
             $classOrInterfaceExists = class_exists($className, false) || interface_exists($className, false);
```

Comment:
For PHP8.1 there should be `enum_exists` call + check everywhere to support enums as well.